nova/virt/libvirt/driver.py

```diff
@@ -5378,6 +5378,8 @@ def get_available_resource(self, nodename):
         data["hypervisor_type"] = self._host.get_driver_type()
         data["hypervisor_version"] = self._host.get_version()
         data["hypervisor_hostname"] = self._host.get_hostname()
+        data["extra_resources"] = self._get_host_sysinfo_serial_hardware()
```

Comment:
Lets add debug logging in both the driver and the compute classes so that it is easy to debug.